client/packages/common/src/intl/strings/IntlStrings.ts

```diff
@@ -1,22 +1,17 @@
 import { useCallback } from 'react';
-import { Namespace, useTranslation as useTranslationNext } from 'react-i18next';
-import { TOptions } from 'i18next';
+import { Namespace, TOptions } from 'i18next';
+import { useTranslation as useTranslationNext } from 'react-i18next';
 import { LocaleKey } from '../locales';
 
+export { TFunction } from 'i18next';
```

Comment:
export TFunction and UseTranslationResponse seem to be unused, I thought it would have a small effect on bundle size increase but doesn't.

Looks like 10% increase is due to: https://github.com/jasonkuhrt/graphql-request/issues/446 (I looked at bundle analyser in 'static' view and found that graphql is much bigger, quick search of bundle size in their repo revealed the above).

I guess this is ok, as long as we are aware of it

Reply:
oops. have removed the `TFunction` that was a remnant from an attempted rewrite

Reply:
and bundle size.. yeah. and darn.
I specifically didn't update `graphql-request` because of that bug, as noted in the description. we'd have to sacrifice codegen in order to get the bundle size down. Or at least manually update all of the generated files.

I've tried the manual approach and the bundle is 2.24MB what do you think? shall we stick with it? Just means that running `yarn generate` will break things until we can get graphql-request updated

Reply:
put the change in 1c2d1ad81 😉